draft-ietf-jsonpath-base.md

```diff
@@ -1193,6 +1193,23 @@ Some examples:
 | `{"a":{"b":{5},c:0}}` | `$[?@.b==5 && !@.c]` | `[{"b":{5},c:0}]` | Existence  |
 
 
+#### Examples
+{: unnumbered}
+
+JSON document:
+
+    [3, 5, 1, 2, 4, 6, {"a": "ij"}, {"a": "ik"}]
+
```

Comment:
We probably also need another set of examples based on a JSON object ... perhaps ...
```
JSON document

{ "a": 1, "b": 2, "c": 3, "d": 5, "e", 7, "u": {"a": 1, "b": 2, "c": 3, "d": 5, "e", 7} }

Some Queries:
   [?@ >1 && @ < 4]
   [?@*.a || @.x]
   ...
```
Note, that we can access first five members via their values only. Index access would require ...

```
[?@index =~ "[b,c,d]"
```
or something like that ...

Reply:
Ok, good suggestion. However `[?@*.a || @.x]` looks (a) syntactically invalid because of `@*` (did you mean `@.*?`) and (b) a bit strange because the `|| @.x` doesn't really add anything of interest (or was that the point?).

Reply:
... no simple typo. `[?@.a || @.x]` is meant to address existence only.

Reply:
@goessner See b8d73680cc579ed757ee142c84364b1d3fae231e.